examples/neutron.rs

```diff
@@ -124,5 +126,40 @@ fn main() -> Result<(), Box<dyn Error>> {
         .with_liq_token_receiver(ACC_0_ADDR)
         .send()?;
 
+    let factory_contract_code_id = ctx
+        .get_contract("astroport_whitelist")
+        .unwrap()
+        .code_id
+        .unwrap();
+
+    // Instantiate a contract predictably
```

Comment:
very minor nit: can we rephrase to // Instantiate a contract with a predictable address

Reply:
Done!